sdk/cdn/Microsoft.Azure.Management.Cdn/src/Properties/AssemblyInfo.cs

```diff
@@ -9,7 +9,7 @@
 [assembly: AssemblyDescription("Provides management functionality for Microsoft Azure CDN.")]
 
 [assembly: AssemblyVersion("4.0.0.0")]
```

Comment:
Please update `AssemblyVersion` as well since it's a major version update

Reply:
Done